source/common/sink_impl.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "nighthawk/common/sink.h"
+
+namespace Nighthawk {
+
+class FileSinkImpl : public Sink {
```

Comment:
Can we add a class comment explaining specifics of this implementation?

Reply:
Done in 8f8a5da